lib/orderbook/MatchingEngine.ts

```diff
@@ -152,11 +158,14 @@ class MatchingEngine {
       ...this.queues.sell.removeMany(callback),
     ] as StampedPeerOrder[];
 
-    // remove from lists. for further optimization, we can maintain a separate list for each peer pubKey
-    removedOrders.forEach((order: StampedPeerOrder) => {
-      const list = order.isBuy ? this.peerOrders.buy : this.peerOrders.sell;
-      list.delete(order.id);
-    });
+    // remove from lists.
+    const peerOrdersList = this.peersOrders.get(peerPubKey);
```

Comment:
I think we can just do `this.peersOrders.delete(peerPubKey)` in stead of looping through and deleting them one by one, right?